README.md

```diff
@@ -58,35 +55,13 @@ Then, in your `BUILD` file:
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
 ```
 
-## Hermetic Python
-
-To configure `pybind11_bazel` for hermetic Python, `python_configure` can take
-the target providing the Python runtime as an argument:
-
-```starlark
-python_configure(
-  name = "local_config_python",
-  python_interpreter_target = "@python_interpreter//:python_bin",
-)
-```
-
 ## Bzlmod
 
 In your `MODULE.bazel` file:
 
 ```starlark
-python_configure = use_extension("@pybind11_bazel//:python_configure.bzl", "extension")
-use_repo(python_configure, "local_config_python", "pybind11")
-```
-
-The `toolchain` tag can only be used by the root module (that is, not by a
-module which is being used as a dependency) to set `python_version` or
-`python_interpreter_target`. For example:
-
-```starlark
-python_configure = use_extension("@pybind11_bazel//:python_configure.bzl", "extension")
-python_configure.toolchain(python_version = "3")
-use_repo(python_configure, "local_config_python", "pybind11")
+pybind11_configure = use_extension("@pybind11_bazel//:pybind11_configure.bzl", "pybind11_configure_extension")
```

Comment:
Something to be aware of is this: https://bazel.build/external/extension#extension_identity

Basically, the label and name you make users use in the use_extension() call is not just a public API, but it is the actual _object identity_ Bazel uses for your extension object. Even if, internally, you refactor things so that two files export the same object, it doesn't matter -- bzlmod will see them as different because it doesn't look any deeper than the label passed to the use_extension() call when defining the object identity. Changing it is then a breaking change to your users.

So, my main point is: make sure you're happy with this particular path and name, because you don't really get to change it.

My second main point is: [one module extension per file](https://bazel.build/external/extension#put_each_extension_in_a_separate_file) (you're doing this already, but I just wanted to reiterated the point; it lets you do more advanced things later if you want to, which you'd otherwise be precluded from).

My third point is just about convention: the convention is to create a `extensions` directory and put each `module_extension` is a separate file. This just collects all the "bzlmod extension stuff" in one place, and tends to help the second point stay true.

All this advice is currently being followed, but I just wanted to reiterate some of the importance and you-only-get-to-choose-once nature of it.

Reply:
Understood. Just to be clear, `pybind11_configure.bzl` is a transitional measure to keep things workable (albeit not ideal) while I remove local_config_python and then proceed as https://github.com/pybind/pybind11_bazel/issues/71#issuecomment-1939537544 describes. In particular, I **_really_** don't want to cut a release while `pybind11_configure.bzl` still exists. This PR is the first of at least a couple to get where we want to be.